_posts/2020-04-20-schedule-your-functions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "On time - schedule your OpenFaaS functions"
+description: "In this post Martin will show you how to deploy a component which will execute your functions on a schedule"
+date: 2020-04-20
+image: /images/2020-04-20-schedule-your-functions/accuracy-alarm-clock-analogue.jpg
+categories:
+  - cronjobs
+  - kubernetes
+  - serverless
+  - linux
+  - tutorial
+  - example
+author_staff_member: martin
+dark_background: true
+
+---
+
+In this blog post I will show you how you can run your functions on schedule. We will first set our environment and then we will run the cron-connector with example function which will be executed on a schedule.
```

Comment:
This line should be the same as "description" IMHO. Don't mind which is which, that's up to you, but make them the same.

Reply:
The description replaces this line now :+1: